sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py

```diff
@@ -28,6 +29,7 @@ class ScheduleSchema(ResourceSchema):
             ArmStr(azureml_type=AzureMLResourceType.JOB),
             CreateJobFileRefField,
             NestedField(PipelineCreateJobSchema),
```

Comment:
What types of jobs are allowed in `CreateJobFileRefField`, e.g. is referencing a sweep job from file allowed?

Reply:
Also curios if backend support specifying sweep job arm id here

Reply:
Yes, the validation check will not perform here, will do it in `_to_rest_object`